fflas-ffpack/fflas/fflas_simd/simd128_int64.inl

```diff
@@ -687,38 +687,40 @@ INLINE CONST vect_t Simd128_impl<true,true,true,8>::mulhi_fast(vect_t x, vect_t
     return x1;
 }
 
-// warning : may be off by 1 multiple, but we save a mul...
-template <bool overflow, bool poweroftwo, int8_t shifter>
-INLINE CONST vect_t Simd128_impl<true,true,true,8>::mod(vect_t &C, const vect_t &P, const vect_t &magic, const vect_t &NEGP,
-                                                        const vect_t &MIN, const vect_t &MAX, vect_t &Q, vect_t &T) {
-#ifdef __INTEL_COMPILER
-    // Works fine with ICC 15.0.1 - A.B.
-    // #warning "not tested"
-    C = _mm_rem_epi64(C, P);
+// FIXME why cannot use Simd128<double>::vect_t in the declaration instead of __m128d?
```

Comment:
I have the same question? is it just not possible or did you just prefer the simpler call to __m128d ?

Reply:
I just couldn't find the right syntax to make it work, but I'm probably just bad at C++.

Reply:
Ok, neither can I after a 5min attempt. Fine to let it as is.